Blog-projects/Widget-window-in-Qt-Quick-app/fontcontrolswidgetsform.cpp

```diff
@@ -0,0 +1,48 @@
+#include "fontcontrolswidgetsform.h"
+#include "ui_fontcontrolswidgetsform.h"
+
+FontControlsWidgetsForm::FontControlsWidgetsForm(QWidget *parent)
+    : QWidget(parent)
+    , ui(new Ui::FontControlsWidgetsForm)
+{
+    ui->setupUi(this);
+    connect(ui->lineEdit, &QLineEdit::textEdited, this, &FontControlsWidgetsForm::lineEditTextChanged);
```

Comment:
Rename the slot to `lineEditTextEdited`, otherwise when I read the slot I thought it was connected to `textChanged` --- but see below, maybe connecting to textChanged would actually be better.

Reply:
Went with the second option and connected straight to the emitter.
Did this for `currentFontChanged` as well.